### ISSUE TITLE
Request Picviewer CE+ support for articulo.mercadolibre.com.mx

### ISSUE THREAD

**Issue by safereg** [hoothin/UserScripts#608]
**The url of site**
The example urls of site. Do not paste only domain without pathname.

https://articulo.mercadolibre.com.mx/MLM-2224812788

Its main idea is to change the "-F" at the end of the small image file name to "-O". I tried using a regular expression replacement rule, but couldn't achieve the desired effect.

**Describe the result you'd like with the rule**
Find the original high-resolution image and show it, like amazon.

**Custom rule you've tried**
 {
        name: "amazon",
        url: /^https?:\/\/http2\.mlstatic\.com\//,
        lazyAttr: "data-a-hires",
        r: /(.+)-F\.webp/,
        s: "$1-O.webp"
 }

**Comment by hoothin** [comment]
```
    {
        "name": "mercadolibre",
        "url": "^https?://articulo\\.mercadolibre\\.com\\.mx/",
        "r": "/(.+)-F\\.webp/i",
        "s": "$1-O.webp"
    }
```
Here is the rule.

**Comment by safereg** [comment]
Thank you for your reply. I tried it but this rule seems not to have taken effect. It did not create a view larger image button like Amazon's.

**Comment by hoothin** [comment]
Did you import the rule by clicking the rules rather than copy-pasting? And which specific image were you referring to?

**Comment by safereg** [comment]
Yes, I clicked on the JSON you provided and the program automatically popped up. I clicked on the confirmation button. However, on the product details page of Mercado, you can see that the original icon does not appear, but it does on Amazon. Thank you very much for your help.

![Amazon](https://github.com/hoothin/UserScripts/assets/171806235/da45d570-a289-469e-ae70-20dc88e7bdb0)

![Mercado](https://github.com/hoothin/UserScripts/assets/171806235/5c30068a-28be-406c-8915-d71a00ea4c69)

**Comment by hoothin** [comment]
![image](https://github.com/hoothin/UserScripts/assets/5361213/d98a7bb3-ed48-4962-a421-b5c77310dd00)

Here is the test result on my end. Make sure you've updated the script to the latest version and that the rules have been successfully imported. (You can turn on the debug switch in the settings and check the console.)
